examples/hc.conf

```diff
@@ -8,7 +8,7 @@ host3
 # Do a sanity check before proceding
 [script1]
 action=execute
-file=/usr/share/gdeploy/scripts/grafton-sanity-check.sh -d disk1,disk2 -h host1,host2
+file=/usr/share/ansible/gdeploy/scripts/grafton-sanity-check.sh -d disk1,disk2 -h host1,host2
```

Comment:
@sac I would suggest you to take this changes in and change the path when you change the path in spec file.